app/src/main/java/com/github/ybecker/epforuml/database/DatabaseManager.kt

```diff
@@ -18,10 +18,23 @@ object DatabaseManager {
     fun getDatabase(): Database {
         return db
     }
+
     /**
      * Modify the current instance of the Database to a MockDatabase
      */
     fun useMockDatabase() {
         db = MockDatabase()
     }
+
+    /**
+     * Synchronizes the local user with the database user.
+     * Should be used when modifying the local user.
+     */
+    fun syncUserWithDatabase() {
+        user.let {
+            if (it != null) {
+                db.updateUser(it)
```

Comment:
Please don't replace whole user in the database. There is some cases when the user in the DatabaseManager is not the same as the one in the database and the method would overwrite some data !